test/e2e/utils/olm.go

```diff
@@ -39,13 +41,15 @@ func (k TestKubernetes) CleanupOLMTest(t *testing.T, subName, subNamespace, subP
 		}
 
 		if panicVal != nil {
-			k.PrintAllResources(subNamespace, &coreosv1.OperatorGroupList{}, map[string]string{})
-			k.PrintAllResources(subNamespace, &coreos.SubscriptionList{}, map[string]string{})
-			k.PrintAllResources(subNamespace, &coreos.ClusterServiceVersionList{}, map[string]string{})
+			dir := os.TempDir() + "/infinispan-operator/" + TestName(t)
+
+			k.PrintAllResources(dir, subNamespace, "OperatorGroup", &coreosv1.OperatorGroupList{}, map[string]string{})
+			k.PrintAllResources(dir, subNamespace, "OperatorGroup", &coreos.SubscriptionList{}, map[string]string{})
+			k.PrintAllResources(dir, subNamespace, "OperatorGroup", &coreos.ClusterServiceVersionList{}, map[string]string{})
 			// Print 2.1.x Operator pod logs
-			k.PrintAllResources(subNamespace, &corev1.PodList{}, map[string]string{"name": "infinispan-operator"})
+			k.PrintAllResources(dir, subNamespace, "OperatorGroup", &corev1.PodList{}, map[string]string{"name": "infinispan-operator"})
 			// Print latest Operator logs
-			k.PrintAllResources(subNamespace, &corev1.PodList{}, map[string]string{"app.kubernetes.io/name": "infinispan-operator"})
+			k.PrintAllResources(dir, subNamespace, "OperatorGroup", &corev1.PodList{}, map[string]string{"app.kubernetes.io/name": "infinispan-operator"})
```

Comment:
The suffixies need some attention here, they are all "OperatorGroup" atm.

Reply:
Updated, thanks